src/components/label/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, h, Prop } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-label",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyLabel {
+	@Prop() color: string
+	@Prop() name: string
+	@Prop() description: string
+	@Prop({ reflect: true }) shape: "rectangle" | "rounded"
+
+	render() {
+		return (
+			<div
+				title={this.description}
+				style={{
+					backgroundColor: `hsl(${this.color} 85% 70%)`,
+				}}>
+				{this.name}
```

Comment:
If the name is just a string is it preferable to pass it using a default slot instead? then the user can chose to pass any form of element inside and not just raw string. would be useful to for example pass `<smoothly-displays>` as inline elements or any other element

Reply:
\+